src/main/java/com/amazon/opendistroforelasticsearch/sql/domain/Condition.java

```diff
@@ -58,6 +58,7 @@
         TERM,
         IDS_QUERY,
         NESTED_COMPLEX,
+        NE_NESTED_COMPLEX, // NOT EXISTS NESTED_COMPLEX
```

Comment:
In most query languages `NE` stands for Not Equals, so although it's a bit longer, but I think `NOT_EXISTS_NESTED_COMPLEX` will be better.

Reply:
Agree. Fixed.